hubblestack/extmods/modules/win_pulsar_winaudit.py

```diff
@@ -415,11 +432,10 @@ def _pull_events(time_frame, checksum):
                     k, v = item.split(':', 1)
                     event_dict[k.strip()] = v.strip()
             #event_dict['Accesses'] = _get_access_translation(event_dict['Accesses'])
-            event_dict['Hash'] = _get_item_hash(event_dict['Object Name'], checksum)
```

Comment:
we will caculate the hash of only those files which are to be monitored as per our pulsar config file